src/init.cpp

```diff
@@ -394,6 +394,7 @@ std::string HelpMessage(HelpMessageMode mode)
     }
     strUsage += HelpMessageOpt("-shrinkdebugfile", _("Shrink debug.log file on client startup (default: 1 when no -debug)"));
     strUsage += HelpMessageOpt("-testnet", _("Use the test network"));
+    strUsage += HelpMessageOpt("-requirestandard", _("Require transactions to be standard even in testnet/regtest"));
```

Comment:
I didn't want to make setting it on mainnet cause any issues, as it's annoying to have scripts and bitcoin.conf files where changing from mainnet to testnet requires more than a single flag to be set; I think the help text as-is gives the impression sufficiently. After all, the fact that it *doesn't* have the default to me at least implies it's an option that only does something when set, and that setting it to =0 off is equivalent to not specifying it at all.